CHANGES

```diff
@@ -73,6 +73,8 @@ Features added
 Bugs fixed
 ----------
 
+* #11093: LaTeX: fix build warnings when one or more reST labels directly preceed an
+  :rst:dir:`py:module` or :rst:dir:`automodule` directive. Patch by Bénédikt Tran (picnixz)
```

Comment:
Can you please change this to reflect the edited title of the pull request and reflow to not exceed 85 chars per line.

Also, please change commit message to "Update CHANGES for PR #11333"

Reply:
And move this after next one so that issue numbers are ordered

Reply:
Actually, I have a question about this. It appears that LaTeX are sorted, but where should I put fixes related to other components ? (for previous versions, it appears that there are no real order...)

Reply:
Yes the order used to be simply that entries where added chronologically, then at some point we started grouping together entries according to builder, and then more recently still I started ordering the LaTeX entries, and only them, as at some point a few weeks ago I did lots of LaTeX work in view of 6.2.0, this felt good.

Reply:
> Yes the order used to be simply that entries where added chronologically,

Sorry for added post-merge noise but to clarify I meant here that new entries used to be added below older ones so the changes are predominantly listed from earliest to latest from top to bottom at each release.